lib/src/revset.rs

```diff
@@ -1691,6 +1719,11 @@ where
             let count = *count;
             RevsetExpression::Latest { candidates, count }.into()
         }
+        RevsetExpression::HasSize { candidates, count } => {
+            let candidates = folder.fold_expression(candidates)?;
+            let count = *count;
```

Comment:
nit: inline this variable for consistency with the other similar uses in this commit